src/java.base/share/classes/java/lang/StableValue.java

```diff
@@ -115,13 +115,13 @@
  *}
  * <p>
  * The {@code getLogger()} method calls {@code logger.orElseSet()} on the stable value to
- * retrieve its content. If the stable value is <em>unset</em>, then {@code orElseSet()}
- * evaluates the given supplier, and sets the content to the result; the content is then
+ * retrieve its contents. If the stable value is <em>unset</em>, then {@code orElseSet()}
+ * evaluates the given supplier, and sets the contents to the result; the contents is then
```

Comment:
@minborg Technically, isn't the *result* returned to the client (after setting it as the contents).

Reply:
Yeah. I will update that, even though it might be an academic question.